### PR TITLE
Archive SYSTEM level easyconfigs from before 2018-10-11

### PR DESCRIPTION
**Pull request by Micket**
I expect to find a few things to revert here in the first go. Letting the CI figure it out.

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/70a6254b9fa366253594ac50cb9f5a92 for a full test report.

File: easybuild/easyconfigs/c/Cereal/Cereal-1.3.0.eb [hunk 1]
```diff
@@ -16,15 +16,20 @@ used standalone."""
...
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+
 configopts = '-DJUST_INSTALL_CEREAL=ON -DSKIP_PERFORMANCE_COMPARISON=ON '
 
 sanity_check_paths = {
     'files': ['include/cereal/cereal.hpp'],
-    'dirs': ['include', 'share'],
+    'dirs': ['include', 'lib/cmake'],
```
Comment (branfosj) [inline easybuild/easyconfigs/c/Cereal/Cereal-1.3.0.eb:32]:
```suggestion
    'dirs': ['include', 'share/cmake/cereal'],
```

That is where that turns up for me.

Comment (Micket) [reply easybuild/easyconfigs/c/Cereal/Cereal-1.3.0.eb]: (I just guessed it was going to be the same as for 1.3.2)

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/2688e64f2e08eb10a0e2082f1c43c632 for a full test report.

**Review by branfosj — APPROVED** [review]

**Pull request merged**